components/form/ApplicantForm.tsx

```diff
@@ -1,61 +1,155 @@
+import { SubmitHandler, useForm, Validate } from 'react-hook-form';
+
+import { MIN_AGE_FOR_APPLICANT } from '../../constants';
+import type ApplicantProperties from '../../types/ApplicantProperties';
+import Alert from '../common/Alert';
 import PrivacyPolicyButtonModal from '../privacy-policy/PrivacyPolicyButtonModal';
 // https://getbootstrap.com/docs/5.0/forms/floating-labels/
 // https://getbootstrap.com/docs/5.0/forms/layout/
 // https://getbootstrap.com/docs/5.0/layout/gutters/
 
-export default function ApplicantForm({ onSubmit, loading }: { onSubmit: (event: React.SyntheticEvent) => void; loading: boolean }): JSX.Element {
+// eslint-disable-next-line max-lines-per-function
+export default function ApplicantForm({ onSubmit, loading, error }: { onSubmit: SubmitHandler<ApplicantProperties>; loading: boolean; error: boolean }): JSX.Element {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<ApplicantProperties>({ mode: 'onTouched' });
+
+  // eslint-disable-next-line no-magic-numbers
+  const YEAR_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 365.2425;
+  const EMAIL_VALIDATION_REGEX =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  const SPECIAL_CHARACTERS = /[\^!#$%*=<>;{}"]/; // https://documentation.onfido.com/#forbidden-characters
```

Comment:
If you look at how many places in this repo has `^!#$%*=<>;{}"` (including tests, error messages, etc), I wonder if it would be good to export as a const `^!#$%*=<>;{}"` and then just import that everywhere (including somehow creating this regex dynamically) rather than redefining that string all the time.